web/massastation/src/pages/Store/StationSection/StationPlugin.tsx

```diff
@@ -80,9 +82,11 @@ export function StationPlugin({
     subtitleIcon: massalabsNomination.includes(author) ? <Certificate /> : null,
     content: [
       updatable && (
-        <Button variant="icon">
+        <Button variant="icon" disabled={isExecuteLoading}>
           <FiRefreshCcw
-            className="text-s-warning"
+            className={`text-s-warning${
+              isExecuteLoading ? ' animate-spin ' : ''
```

Comment:
so this is a tailwind preset, unfortunately it only spins clock wise, when we need to spin our icon counter clock wise. The result is weird. We have two option : flip the icon or add a custom tailwind class se [here](https://github.com/tailwindlabs/tailwindcss/discussions/2220#discussioncomment-596742)  @Pierregvx  @mazmassa wdyt ?

Reply:
[Screencast from 27-06-2023 11:29:34.webm](https://github.com/massalabs/station/assets/83134981/3c326d28-e73b-4c21-954e-1604bbc6c5ad)
better ?

Reply:
perfect ! does it spin for the whole update duration ?

Reply:
yes sir

Reply:
Are you disabling the button too!?

Reply:
it should be disabled but there is a problem with it

Reply:
fixed